### PR TITLE
Catch exception when check state in StateWheelExecuteThread

### PR DESCRIPTION
**Pull request by ruanwenjun**
## Purpose of the pull request

- Query active taskInstance from taskInstanceMap rather than processor, since when refresh taskInstance in process may loss some fields, e.g taskDefinition

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Review by kezhenxu94 — DISMISSED** [review]

File: dolphinscheduler-standalone-server/src/main/resources/application.yaml [hunk 1]
```diff
@@ -134,7 +134,7 @@ master:
   # master heartbeat interval
   heartbeat-interval: 10s
   # master commit task retry times
-  task-commit-retry-times: 5
+  task-commit-retry-times: 5s
```
Comment (caishunfeng) [inline dolphinscheduler-standalone-server/src/main/resources/application.yaml]: :eyes:

File: dolphinscheduler-standalone-server/src/main/resources/application.yaml [hunk 2]
```diff
@@ -134,7 +134,7 @@ master:
...
   # master commit task interval
   task-commit-interval: 1s
   state-wheel-interval: 5
```
Comment (caishunfeng) [inline dolphinscheduler-standalone-server/src/main/resources/application.yaml]: you should change this one

Comment (ruanwenjun) [reply dolphinscheduler-standalone-server/src/main/resources/application.yaml]: Thanks for your review, this is a mistake.

**Review by caishunfeng — APPROVED** [review]
LGTM

**Pull request merged**